package.json

```diff
@@ -75,8 +74,8 @@
     "mt-changelog": "^0.6.1",
     "node-libs-browser": "^0.5.2",
     "raw-loader": "^0.5.1",
-    "react": "^0.13.3",
-    "react-bootstrap": "^0.24.2",
```

Comment:
you should be able to use ` "react-bootstrap": "0.24.5-react-pre.0"` here its the react 014 branch